src/components/Gradebook/index.jsx

```diff
@@ -379,20 +406,32 @@ export default class Gradebook extends React.Component {
                       options={[{ label: 'Percent', value: 'percent' }, { label: 'Absolute', value: 'absolute' }]}
                       onChange={this.props.toggleFormat}
                     />
-                    {this.props.assignmentTypes.length > 0 &&
-                      <div className="student-filters">
-                        <span className="label">
-                          Assignment Types:
-                        </span>
-                        <InputSelect
-                          name="assignment-types"
-                          ariaLabel="Assignment Types"
-                          value={this.props.selectedAssignmentType}
-                          options={this.mapAssignmentTypeEntries(this.props.assignmentTypes)}
-                          onChange={this.updateAssignmentTypes}
-                        />
-                      </div>
-                    }
+                    <div className="student-filters">
+                      <span className="label">
+                        Assignment Types:
+                      </span>
+                      <InputSelect
+                        name="assignment-types"
+                        ariaLabel="Assignment Types"
+                        value={this.props.selectedAssignmentType}
+                        options={this.mapAssignmentTypeEntries(this.props.assignmentTypes)}
+                        onChange={this.updateAssignmentTypes}
+                        disabled={this.props.assignmentFilterOptions.length === 0}
+                      />
+                    </div>
+                    <div className="student-filters">
+                      <span className="label">
+                        Assignment:
+                      </span>
+                      <InputSelect
+                        name="assignment-types"
+                        ariaLabel="Assignment Types"
```

Comment:
I think that the `ariaLabel` prop should be `Assignment` and the `name` prop `assignment`. Was this a copy-paste error, or am I overlooking something?